CMakeLists.txt

```
@@ -53,6 +53,7 @@ if (NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE "Release" CACHE STRING
     "Build type from [Debug, Release]. For perf testing, build Release" FORCE)
```

This is outdated, now it's:
```suggestion
    "Build type from [Debug, DevDebug, Release, RelWithDebInfo]. For perf testing, build Release" FORCE)
```

---

True, it can be just called `CMAKE_BUILD_TYPE`.